### PR TITLE
LautFM Presence

### PR DESCRIPTION
**Pull request by MauriceX24**
Discord_dFk0dJsvRo

### CODE REVIEW

File: LautFM/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "version": "1.0",
+    "logo": "https://i.imgur.com/rncpOUV.png",
+    "thumbnail": "https://i.imgur.com/FZh7OFc.png",
+    "color": "#373b4a",
+    "tags": [
+        "Music",
+		"Radio",
+		"Station"
+    ],
+    "iframe": false,
```
Comment (frozaum) [inline LautFM/dist/metadata.json]: iframe not required

File: LautFM/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "service": "LautFM",
+    "description": {
+        "en": "Laut.FM is a Website to create your own Music station or listening to music.",
+        "de": "Laut.FM ist eine Webseite um dein eigenes Radio zu erstellen oder um Musik zu hören."
+
+    },
+    "url": [
+        "laut.fm"
+
```
Comment (frozaum) [inline LautFM/dist/metadata.json]: remove this blank space

File: LautFM/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "author": {
+        "name": "MauriceX",
+        "id": "268768664411439115"
+    },
+    "service": "LautFM",
+    "description": {
+        "en": "Laut.FM is a Website to create your own Music station or listening to music.",
+        "de": "Laut.FM ist eine Webseite um dein eigenes Radio zu erstellen oder um Musik zu hören."
+
```
Comment (frozaum) [inline LautFM/dist/metadata.json]: remove this blank space

File: LautFM/dist/metadata.json (hunk 1 above)
Comment (MauriceX24) [reply LautFM/dist/metadata.json]: Iframe was removed and the blank space was removed too.

**Pull request merged**